ynr/apps/parties/constants.py

```diff
@@ -0,0 +1,51 @@
+EC_API_BASE = (
+    "http://search.electoralcommission.org.uk/api/search/Registrations"
+)
+EC_EMBLEM_BASE = (
+    "http://search.electoralcommission.org.uk/Api/Registrations/Emblems"
+)
+
+DEFAULT_EMBLEMS = {
```

Comment:
Could we have a comment explaining why we need this - why do these parties need this, why don't others?